### ISSUE TITLE
APP THEME: Button missing a text alternative in search component in application theme - 1 page with this issue

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#715]
Siteimprove report -
https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r12&ruleId=12&issueKind=1&tagFilterType=2&exceptTags=1,2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2
